sdk/spring/CHANGELOG.md

```diff
@@ -1,49 +1,169 @@
 # Release History
 
 ## 4.0.0-beta.4 (Unreleased)
-- Support Spring Boot version: 2.5.5+ and 2.6.0 - 2.6.3.
-- Support Spring Cloud version: 2020.0.3+ and 2021.0.0.
+- Support Spring Boot version: 2.5.0+ and 2.6.0+. (Note: 2.5.11+ and 2.6.5+ should be supported, but they aren't tested with this release.)
```

Comment:
The problem still exists:
 - 2.6.0+ is confusing. Do these versions belong to 2.6.0+: `2.6.1`, `2.6.11`, `2.7.0`, `3.0.0`?

How about do like this:

> Support Spring Boot version: 2.5.x(0<=x<=10) and 2.6.y(0<=y<=5). **Note**: 2.5.x(x>10) and 2.6.y(y>5) should be supported, but they aren't tested with this release